reflectable/lib/src/builder_implementation.dart

```diff
@@ -4109,10 +4118,14 @@ class BuilderImplementation {
     }
     imports.sort();
 
+    String languageVersionComment =
+        world.entryPointLibrary.isNonNullableByDefault
+            ? ''
```

Comment:
Should we add a `@dart = 2.12` to this case to future proof?

Reply:
I'm not sure: If someone is using old tools then language version 2.12 may not exist (but they may have been using `--enable-experiment=non-nullable`, and we don't have to break them).

The other direction should be safe: If we specify language version 2.12 and the tools and client package are at 2.17 then it _ought_ to be working (because it takes version ^3.0.0-0 to break), but the introduction of null safety shows that it's nowhere near safe to jump silently into a version 2.x.y.

So I added it.